src/utils/preconditions.ts

```
@@ -18,23 +20,74 @@ export async function checkChannelRights(interaction: ButtonInteraction | Comman
 export function CheckRights(): Gate<GateContextBase, "CheckRights"> {
     return defineGate("CheckRights", async (ctx) => {
         const userId = ctx.userId
+        if (!userId) throw new NoRights("manager")
         const channel = await database.findChannel(ctx.channelId!)
-        if (channel?.ownerId != userId) throw new NoRights()
+        const isManager = Array.isArray(channel?.managers) && channel.managers.includes(userId)
+        if (channel?.ownerId != userId && !isManager) throw new NoRights("manager")
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)